src/Sulu/Bundle/ContactBundle/Resources/public/js/components/contacts/components/list/decorators/contact-view.js

```diff
@@ -0,0 +1,261 @@
+/**
+ * @class ThumbnailView (Datagrid Decorator)
+ * @constructor
+ *
+ * @param {Object} [viewOptions] Configuration object
+ *
+ * @param {Boolean} [rendered] property used by the datagrid-main class
+ * @param {Function} [initialize] function which gets called once at the start of the view
+ * @param {Function} [render] function to render data
+ * @param {Function} [addRecord] function to add a new record to the grid
+ * @param {Function} [removeRecord] function to remove an existing record from the grid
+ * @param {Function} [destroy] function to destroy the view and unbind events
+ */
+define(function() {
+
+    'use strict';
+
+    var defaults = {
+            unselectOnBackgroundClick: true,
+        },
+
+        constants = {
+            containerClass: 'contact-grid',
+            selectedClass: 'selected',
+            itemHeadClass: 'item-head',
+            itemInfoClass: 'item-info',
+
+            idProperty: 'id',
+            mailProperty: 'mainEmail',
+        },
+
+        templates = {
+            item: [
+                '<div class="contact-item">',
+                '   <div class="' + constants.itemHeadClass + '">',
+                '       <div class="head-container">',
+                '           <div class="image" style="background-image: url(\'<%= picture %>\')"></div>',
+                '           <div class="head-name"><%= name %></div>',
+                '       </div>',
+                '       <div class="head-checkbox custom-checkbox"><input type="checkbox"><span class="icon"></span></div>',
+                '       <% if (!!isSuluUser) { %>',
+                '       <div class="head-sulubox"></div>',
+                '       <% } %>',
+                '   </div>',
+                '   <div class="' + constants.itemInfoClass + '">',
+                '       <% if (location !== "undefined") { %>',
+                '       <div class="info-row">',
```

Comment:
@nnatter could you make an own template 'infoRow' and append the to rows via javascript?